ow_lander/config/ros_controllers.yaml

```diff
@@ -211,4 +178,15 @@ grinder_controller:
       p: *kp_default
       d: *kd_default
       i: *ki_default
-      i_clamp: *ki_clamp_default
\ No newline at end of file
+      i_clamp: *ki_clamp_default
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 300
```

Comment:
Did you change the publish_rate intentionally? what is the reason for this change?

Reply:
Oh I put it equal to the control loop frequency, but maybe they don't have to be equal. What's the reason why we choose 50?

Reply:
That's the default value set by **moveit setup_assistant**

Reply:
So I am not very in favor of changing as this may have other side effects.

Reply:
I also recommend keeping the block position in the same place as before. The reason if we every run setup assistant again then it wouldn't change as something that has change.

Reply:
ok